### ISSUE TITLE
False positive Z459 with negative numbers.

### ISSUE THREAD

**Issue by levonikitatech** [wemake-services/wemake-python-styleguide#585]
# Bug report

## What's wrong

This __init__ method raises Z459, because cluster_index is -1 by default.
```
class FiguresSettings:
    def __init__(
        self,
        cluster_index=-1,
        reed_indexes=DEFAULT_REED_INDEXES,
        treshold=1024,
    ):
        self.cluster_index = cluster_index
        self.reed_indexes = reed_indexes
        self.treshold = treshold
```

## How is that should be
-1 is ast.Num, it shouldn't raise Z459.

### flake8 information

Contents of `flake8 --bug-report`:
`
{
  "dependencies": [
    {
      "dependency": "entrypoints",
      "version": "0.3"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.7.1",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-annotations-complexity",
      "version": "0.0.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "v1.0.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-broken-line",
      "version": "0.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "18.8.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "1.4.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-debugger",
      "version": "3.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.3.0, pydocstyle: 3.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-eradicate",
      "version": "0.2.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-print",
      "version": "3.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.2.3"
    },
    {
      "is_local": false,
      "plugin": "flake8-type-annotations",
      "version": "0.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_builtins",
      "version": "1.4.1"
    },
    {
      "is_local": false,
      "plugin": "flake8_coding",
      "version": "1.3.1"
    },
    {
      "is_local": false,
      "plugin": "flake8_commas",
      "version": "2.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "2.3"
    },
    {
      "is_local": false,
      "plugin": "flake8_pep3101",
      "version": "1.2.1"
    },
    {
      "is_local": false,
      "plugin": "flake8_quotes",
      "version": "1.0.0"
    },
    {
      "is_local": false,
      "plugin": "logging-format",
      "version": "0.6.0"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.7.0"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.5.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.1.1"
    },
    {
      "is_local": false,
      "plugin": "wemake-python-styleguide",
      "version": "0.8.1"
    }
  ],
  "version": "3.7.7"
}
`
### pip information

Contents of `pip freeze`:

`
aiohttp==3.4.4
alabaster==0.7.12
astor==0.7.1
astroid==2.0.4
async-timeout==3.0.1
asyncpg==0.18.2
attrs==18.2.0
autopep8==1.4
Babel==2.6.0
bandit==1.5.1
beautifulsoup4==4.7.1
certifi==2018.10.15
chardet==3.0.4
doc8==0.8.0
docopt==0.6.2
docutils==0.14
entrypoints==0.3
eradicate==0.2.1
flake8==3.7.7
flake8-annotations-complexity==0.0.2
flake8-bandit==1.0.2
flake8-broken-line==0.1.0
flake8-bugbear==18.8.0
flake8-builtins==1.4.1
flake8-coding==1.3.1
flake8-commas==2.0.0
flake8-comprehensions==1.4.1
flake8-debugger==3.1.0
flake8-docstrings==1.3.0
flake8-eradicate==0.2.0
flake8-isort==2.7.0
flake8-logging-format==0.6.0
flake8-pep3101==1.2.1
flake8-polyfill==1.0.2
flake8-print==3.1.0
flake8-quotes==1.0.0
flake8-string-format==0.2.3
flake8-type-annotations==0.1.0
gitdb2==2.0.5
GitPython==2.1.11
hbmqtt==0.9.4
idna==2.7
imagesize==1.1.0
iniherit==0.3.9
isort==4.3.4
Jinja2==2.10.1
jsonschema==2.6.0
lazy-object-proxy==1.3.1
MarkupSafe==1.1.1
mccabe==0.6.1
motor==2.0.0
multidict==4.4.2
packaging==19.0
paho-mqtt==1.4.0
passlib==1.7.1
pbr==5.1.0
pep8-naming==0.7.0
pycodestyle==2.5.0
pydocstyle==3.0.0
pyflakes==2.1.1
Pygments==2.4.0
pylint==2.1.1
pymongo==3.7.2
pyparsing==2.4.0
pytz==2019.1
PyYAML==3.13
requests==2.22.0
restructuredtext-lint==1.3.0
rope==0.11.0
six==1.11.0
smmap2==2.0.5
snowballstemmer==1.2.1
soupsieve==1.9.1
Sphinx==2.0.1
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
stevedore==1.30.0
testfixtures==6.3.0
text-unidecode==1.2
transitions==0.6.8
typing-extensions==3.6.6
urllib3==1.25.2
websockets==6.0
wemake-python-styleguide==0.8.1
wrapt==1.10.11
yarl==1.2.6
yoyo-migrations==6.0.0
`

**Comment by sobolevn** [comment]
Indeed, are you willing to fix it?
I will accept a PR with a great pleasure.

**Comment by levonikitatech** [comment]
@sobolevn It's my mistake. In python3 
`-1` translates into
`UnaryOp(op=USub(), operand=Num(n=1))` witch is not ast.Num

Let's close issue? We can allow unary minus before num.

**Comment by sobolevn** [comment]
Yes, we should allow negative numbers. They are numbers after all!

**Comment by sobolevn** [comment]
I have implemented it in the `0.9.0` release. Sorry for the delay.
